docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb

```diff
@@ -4,30 +4,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Create a custom operator\n",
+    "# Create a Custom Operator\n",
     "\n",
-    "DALI allows you to create a custom operator in C++ and load it at runtime. There are several reasons you might need to write your custom operator, for instance\n",
+    "DALI allows you to create a custom operator in C++ and load it at runtime. Here are several reasons you might need to write your custom operator:\n",
     "\n",
-    "- DALI doesn't support the operation you want to perform and it cannot be expressed by a composition of other operators\n",
-    "- You want to write an operator that depends on a third party library\n",
-    "- You want to optimize your pipeline by providing a manually fused operation in C++\n",
+    "- DALI does not support the operation that you want to perform and it cannot be expressed by a composition of other operators.\n",
+    "- You want to write an operator that depends on a third-party library.\n",
+    "- You want to optimize your pipeline by providing a manually fused operation in C++.\n",
     "\n",
-    "In this tutorial we will walk you through the process of writing, compiling and loading a plugin with a DALI custom operator. For demonstration purposes we will provide a CPU and a GPU implementation for the 'CustomDummy' operator. The implementation will just copy the input data to the output without any modifications.\n",
+    "In this tutorial, we will walk you through the process of writing, compiling, and loading a plugin with a DALI custom operator. For demonstration purposes we will provide a CPU and a GPU implementation for the CustomDummy operator. The implementation only copies the input data to the output without any modifications.\n",
     "\n",
-    "PREREQUISITES:\n",
+    "### Prerequisites\n",
```

Comment:
This level of nesting broke the navigation, it seems that `##` needs to be introduced first.